dist/PathTools/t/crossplatform.t

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/perl -w
 
-use strict;
```

Comment:
same comment, we should not remove `use strict` for dual life modules

Reply:
Noted, see commit messages in updated p.r.